### PR TITLE
add Italian translations

### PR DESCRIPTION
**Pull request by drc0**

### CODE REVIEW

File: i18n/it_IT.json [hunk 1]
```diff
@@ -79,24 +79,24 @@
...
-    "Date": null,
+    "Date": "Data",
     "Default %s address": "Indirizzo predefinito",
     "Delete": "Elimina",
     "Description:": "Descrizione:",
-    "Description: ": null,
+    "Description: ": "Descrizione:",
     "Don't have an account?": "Non hai un account?",
-    "Don`t have an account?": null,
+    "Don`t have an account?": "Non hai un account?",,
```
Comment (Danpiel) [inline i18n/it_IT.json:99]: Codacy Issue found: [Unexpected character (',' (code 44)): was expecting double-quote to start field name](https://app.codacy.com/manual/ScandiPWA/base-theme/pullRequest?prid=5403666)

**Comment by Danpiel** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
- Added 1
           
```


See the complete overview on [Codacy](https://app.codacy.com/manual/ScandiPWA/base-theme/pullRequest?prid=5403666&bid=17713900)

File: i18n/it_IT.json (hunk 1 above)
Comment (drc0) [reply i18n/it_IT.json]: Sorry for that, pushed a fix.

**Comment by alfredsgenkins** [comment]
Thank you for an amazing job done! I will merge this as soon as possible. Currently deciding on whether we should merge it into 2.x or 3.x :) I will add your to contributors as well as soon as I decide with a branch!

**Pull request merged**